src/NuGet.Core/NuGet.Versioning/VersionComparer.cs

```diff
@@ -12,7 +12,9 @@ namespace NuGet.Versioning
     /// </summary>
     public sealed class VersionComparer : IVersionComparer
     {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
         public static IVersionComparer Get(VersionComparison versionComparison)
+#pragma warning restore CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
I think this one is sufficiently simple to document, just from guessing based on symbol names. Using the IDE to see the definition of the VersionComparison type, I think you'll quickly have a good understanding of what this API does, or just read this method's body below.

This PR, as it currently stands, has very few suppressions outside of NuGet.CommandLine, so I'd like to see the analyzer warnings being fixed, rather than suppressed.